easybuild/easyconfigs/q/q2-krona/q2-krona-20220124-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonBundle'
+
+name = 'q2-krona'
+version = '20220124'
+_commit = 'd34c526'
+
+homepage = 'https://library.qiime2.org/plugins/q2-krona/39/'
+description = """QIIME2 plugin for creating Krona plots"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
```

Comment:
I'm confused. If this is a QIIME2 plugin, doesn't it depend on QIIME2?

Reply:
By the way, feel free to ping another maintainer on Slack after you responded to this - I'm going on holiday for a while :)

As long as this was an ommission that was done on purpose, the rest of the PR looks good te me and can be merged.

Reply:
this was on purpose indeed.
the plugin does not depend on a specific version, so can be used with different QIIME2 versions, for example this PR has another version in the same toolchain: https://github.com/easybuilders/easybuild-easyconfigs/pull/19161